upstream-community-operators/postgresql/4.5.1/postgresoperator.v4.5.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,810 @@
+                  value: registry.developers.crunchydata.com/crunchydata
+                - name: PGO_IMAGE_TAG
+                  value: centos7-4.5.1
+                - name: RELATED_IMAGE_PGO_BACKREST
+                  value: registry.developers.crunchydata.com/crunchydata/pgo-backrest:centos7-4.5.1
+                - name: RELATED_IMAGE_PGO_BACKREST_REPO
+                  value: registry.developers.crunchydata.com/crunchydata/pgo-backrest-repo:centos7-4.5.1
+                - name: RELATED_IMAGE_PGO_BACKREST_REPO_SYNC
+                  value: registry.developers.crunchydata.com/crunchydata/pgo-backrest-repo-sync:centos7-4.5.1
+                - name: RELATED_IMAGE_PGO_BACKREST_RESTORE
+                  value: registry.developers.crunchydata.com/crunchydata/pgo-backrest-restore:centos7-4.5.1
+                - name: RELATED_IMAGE_PGO_CLIENT
+                  value: registry.developers.crunchydata.com/crunchydata/pgo-client:centos7-4.5.1
+                - name: RELATED_IMAGE_PGO_RMDATA
+                  value: registry.developers.crunchydata.com/crunchydata/pgo-rmdata:centos7-4.5.1
+                - name: RELATED_IMAGE_PGO_SQL_RUNNER
+                  value: registry.developers.crunchydata.com/crunchydata/pgo-sqlrunner:centos7-4.5.1
+                - name: RELATED_IMAGE_CRUNCHY_POSTGRES_EXPORTER
+                  value: registry.developers.crunchydata.com/crunchydata/crunchy-postgres-exporter:centos7-4.5.1
+                - name: RELATED_IMAGE_CRUNCHY_ADMIN
+                  value: registry.developers.crunchydata.com/crunchydata/crunchy-admin:centos7-12.5-4.5.1
+                - name: RELATED_IMAGE_CRUNCHY_BACKREST_RESTORE
+                  value: registry.developers.crunchydata.com/crunchydata/crunchy-backrest-restore:centos7-12.5-4.5.1
+                - name: RELATED_IMAGE_CRUNCHY_PGADMIN
+                  value: registry.developers.crunchydata.com/crunchydata/crunchy-pgadmin4:centos7-12.5-4.5.1
+                - name: RELATED_IMAGE_CRUNCHY_PGBADGER
+                  value: registry.developers.crunchydata.com/crunchydata/crunchy-pgbadger:centos7-12.5-4.5.1
+                - name: RELATED_IMAGE_CRUNCHY_PGBOUNCER
+                  value: registry.developers.crunchydata.com/crunchydata/crunchy-pgbouncer:centos7-12.5-4.5.1
+                - name: RELATED_IMAGE_CRUNCHY_PGDUMP
+                  value: registry.developers.crunchydata.com/crunchydata/crunchy-pgdump:centos7-12.5-4.5.1
+                - name: RELATED_IMAGE_CRUNCHY_PGRESTORE
+                  value: registry.developers.crunchydata.com/crunchydata/crunchy-pgrestore:centos7-12.5-4.5.1
+                - name: RELATED_IMAGE_CRUNCHY_POSTGRES_HA
+                  value: registry.developers.crunchydata.com/crunchydata/crunchy-postgres-ha:centos7-12.5-4.5.1
+                - name: RELATED_IMAGE_CRUNCHY_POSTGRES_GIS_HA
+                  value: registry.developers.crunchydata.com/crunchydata/crunchy-postgres-gis-ha:centos7-12.4-3.0-4.5.1
```

Comment:
Same, should be `registry.developers.crunchydata.com/crunchydata/crunchy-postgres-gis-ha:centos7-12.5-3.0-4.5.1`